documentation/development/contributing.md

```diff
@@ -0,0 +1,38 @@
+# Contributing to NostrHub
+
+The NostrHub project operates an open contributor model where anyone is welcome to contribute towards development in the form of peer review, documentation, testing and patches.
+Anyone is invited to contribute without regard to technical experience, "expertise", OSS experience, age, or other concern.
+
+# Communications Channels
+
+All communication happens openly, using github issues, PRs, nostr, or the [Telegram group](https://t.me/nostr_protocol). This ensures that hidden information does not exist, which makes contributing much easier. Please post your nostr public key [here](https://github.com/NostrGit/NostrGit/issues/19) to be noticed on nostr by the team.
```

Comment:
the telegram is super crowded, maybe remove it and just here?

Reply:
Was hoping you would say that ;)

Reply:
crowded as in -- hard to keep track of

Reply:
agree

Reply:
man i cant wish for nostr-based group chat